### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by spdkimo**
Using Docker image instruction is added

### CODE REVIEW

**Comment by sangsoolee** [comment]
To make the composition of README items more harmonious, make the docker guide a sub-item of **how to install and execute** and provide a link.

File: README.md [hunk 1]
```diff
@@ -99,3 +99,45 @@ https://github.com/project-chip/connectedhomeip/issues/6347
...
+
+It confirmed that is working on below host OS:
+- Ubuntu 20.04
+
+It also confirmed that there are unsupported host OS:
+- Ubuntu 16.04
+- macOS
+
+### How to start
+Threre are things to be needed to prepare on host side
```
Comment (junyong-sim) [inline README.md:115]: Threre -> There

File: README.md [hunk 2]
```diff
@@ -99,3 +99,45 @@ https://github.com/project-chip/connectedhomeip/issues/6347
 
 For Ioter contribution, see our [Contributing Guidelines](https://github.com/Samsung/ioter/blob/main/CONTRIBUTING.md) for more information.
 We welcome your contribution at any time.
+
+
+## Docker image support
+You can use a Docker image that 'ioter' environment is prepared
```
Comment (junyong-sim) [inline README.md:105]: There is no '.' at the end of sentence.

**Review by spdkimo — COMMENTED** [review]
Apply all requested changes

**Review by junyong-sim — COMMENTED** [review]
LGTM

But, I think you need to squash some commits.

git checkout main
git pull upstream main

# If there were any new commits, rebase your development branch
git checkout <branch-name>
git rebase -i main

This will open up a text editor where you can specify which commits to squash.

**Review by junyong-sim — APPROVED** [review]

**Review by junyong-sim — APPROVED** [review]

**Pull request merged**